#!/usr/bin/env python
#   Desc:   Allows parameters etc to be tuned in realtime using dynamic reconfigure
PACKAGE = "slic3r_coverage_planner"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

#        Name                      Type  Level  Description                           Default   Min    Max
gen.add( "perimeter_clockwise",    bool_t, 1, "follow outer perimeter CW instead of CCW", True )
gen.add( "visualize_plan" ,        bool_t, 1, "visualize plan by publishing marker array", True)
gen.add( "costmap_converter",      bool_t, 1, "use costmap_converter to respect global obstacles", True)
                 # PkgName  #NodeName            #Prefix for generated .h include file, e.g. ParametersConfig.py
exit(gen.generate(PACKAGE, "CoveragePlanner", "CoveragePlanner"))